CollisionOperator/scatteringKernels_func.f90

```diff
@@ -113,7 +115,7 @@ function targetVelocity_constXS(E,dir,A,kT,rand) result (V_t)
     Y = sqrt(A*E/kT)
 
     ! Calculate threshold factor alpha
-    alpha = 2.0/(Y*sqrt(PI)+2.0)
+    alpha = TWO/(Y*sqrt(PI) + TWO)
```

Comment:
make sqrt(PI) a universal variable? One would imagine it must turn up elsewhere...